concepts/type-conversion/about.md

```diff
@@ -1,3 +1,43 @@
-# About
+# About Type Conversion
 
-TODO: add information on type-conversion concept
+In Go, assignment of a value between different types requires explicit conversion.
```

Comment:
I find this sentence confusing because type conversion is not only needed in the context of an assignment but also for comparison for example, or when providing an argument for a function.
```go
var i int8 = 12
var j int32 = 13
fmt.Println(i == j)
// invalid operation: i == j (mismatched types int8 and int32)
```
Here a possible alternative:
```suggestion
Go does not implicitly convert values from one type to another. If you want to use a value of one type in a context where another type is required, you need to explicitly convert the value.
```
(Needs to be fixed in the all the places if you decide to change something here.